libs/lib-client/mapper/src/services/mapper-controller/mapper-controller-factory.service.ts

```diff
@@ -28,22 +27,33 @@ export class MapperControllerFactory {
     input: AttributeDescriptor[],
     output: AttributeDescriptor[],
     mappings: any,
-    functions: InstalledFunctionSchema[]
+    functions: InstalledFunctionSchema[],
+    makeSnippet: (nodes) => string,
+    MapperTranslator
```

Comment:
I think we need to create an interface for MapperTranslator so this can be `mapperTranslator: MapperTranslator` and also update its usage across the app.

In this point this resolves as `mapperTranslator: any` so we're losing type safety here and many other places where the MapperTranslator is used as `any`